class.jetpack.php

```diff
@@ -7376,10 +7376,14 @@ public static function load_jetpack_gutenberg() {
 			plugins_url( '_inc/blocks/', JETPACK__PLUGIN_FILE )
 		);
 
+		$view_script_dependencies = array(
+			'wp-i18n',
```

Comment:
A caution here. If #10328 lands, the locale data is moved to the editor assets which wouldn't be included in the frontend.

We really need a way to localize exactly one time, there are several different ways we could do that.